src/sentry/tasks/statistical_detectors.py

```diff
@@ -466,7 +470,7 @@ def query_functions_timeseries(
     start: datetime,
     agg_function: str,
 ) -> Generator[Tuple[int, int, Any], None, None]:
-    project_ids = [project_id for project_id, _ in functions_list]
+    project_ids = [item[0] for item in functions_list]
```

Comment:
This is in preparation for a signature change where the functions list will become a triple of `(project_id, fingerprint, description)` and the current destructuring would error.